server/models/Request.js

```diff
@@ -1,36 +1,36 @@
 const mongoose = require("mongoose");
 
 const requestSchema = new mongoose.Schema({
-  ownerId: { 
+  ownerId: {
```

Comment:
This should be `user` since you referenced 'User" here

Reply:
Hey Sunday, I think having the field name as `ownerId` should be okay, as each request consists of a sitter and owner (both which refer to the 'user' model) ? Thoughts?